composer.json

```diff
@@ -46,7 +46,7 @@
         "ext-openssl": "*",
         "ext-session": "*",
         "ezyang/htmlpurifier": "^4.17",
-        "henrique-borba/php-sieve-manager": "^1.0",
+        "henrique-borba/php-sieve-manager": "^1.0.6",
```

Comment:
I think the removed code was correct. Can you confirm please @marclaporte ?

Reply:
composer.lock will provide 1.0.6 so it doesn't matter much.

As per http://semver.org/, going from 1.0.5 to 1.0.6 should not require any changes for Cypht. If Cypht requires changes, maybe it should have been 1.1.0.

It's not critical now since the lib is not yet included in many projects. But in the future, let's try to adhere to SemVer for all code susceptible to be leveraged or included by another project.